### PR TITLE
Attempting to add missing code exercises to homeworks

### PR DESCRIPTION
**Pull request by THEJRRR**
The email today notified me that the code exercises were not added to my previous pull request, which I am now trying to correct. My apologies.

### CODE REVIEW

File: week3/homework/calculator.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+			return pow((1/x), -n)
+		elsif n == 0 
+		 	return 1
+		elsif n == 1 
+			return x
+		elsif n % 2 == 0 
+			return pow((x*x), (n/2))
+		else
+			return x * pow((x*x), (n-1)/2)
+		end
```
Comment (reneedv) [inline week3/homework/calculator.rb:36]: There might be some simplification you could do here.

File: week3/homework/calculator.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
+class Calculator
+	def initialize()
+	end 
+
+	def sum(num_array)
+		if num_array.length > 0
```
Comment (reneedv) [inline week3/homework/calculator.rb:6]: there is a way you could use inject where you would not need this if statement.

File: week3/homework/questions.txt [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+A symbol in Ruby is an identifier corresponding to a name, but can be any string of characters. 
+
+2. What is the difference between a symbol and a string?
+
+The object id for a symbol will always be the same, but a string's object id will return different objects. 
+
+3. What is a block and how do I call a block?
+
+A block is a chunk of code that can be run as the file is loaded. You must call out these blocks with a Begin block and an End block.
```
Comment (reneedv) [inline week3/homework/questions.txt:17]: What do you mean as a file is loaded?

**Comment by reneedv** [comment]
Good work!

**Comment by THEJRRR** [comment]
I wanted to try a self referencing function, and this was an opportunity to
try it. I can appreciate it being a little too cute for the exercise.

On Thu, Jan 30, 2014 at 5:27 PM, Renée Hendricksen <notifications@github.com

**Pull request closed without merging**